README.md

```diff
@@ -70,17 +70,17 @@ Other paths to files should be set according to your configuration, using a file
 
 In one terminal window, open the snap daemon (in this case with logging set to 1,  trust disabled and global configuration saved in config.json ):
 ```
-$ $SNAP_PATH/bin/snapd -l 1 -t 0 --config config.json
+$ $SNAP_PATH/bin/snapteld -l 1 -t 0 --config config.json
```

Comment:
Could you remove $SNAP_PATH from this documentation or correct this line in documentation:
```
$ export SNAP_PATH=<snapDirectoryPath>/build
```
to current localization of snap's binaries?

See this [example](https://github.com/intelsdi-x/snap-plugin-collector-cpu#examples)

If you correct it, please change commit message and tile to of pull request to have full information in them.